opendbc/car/honda/carstate.py

```diff
@@ -52,7 +52,7 @@ def get_can_messages(CP, gearbox_msg):
   if CP.flags & HondaFlags.BOSCH_ALT_BRAKE:
     messages.append(("BRAKE_MODULE", 50))
 
-  if CP.carFingerprint in (HONDA_BOSCH | {CAR.HONDA_CIVIC, CAR.HONDA_ODYSSEY, CAR.HONDA_ODYSSEY_CHN}):
+  if (CP.carFingerprint in (HONDA_BOSCH | {CAR.HONDA_CIVIC, CAR.HONDA_ODYSSEY, CAR.HONDA_ODYSSEY_CHN})) and CP.carFingerprint != CAR.HONDA_NBOX:
```

Comment:
```suggestion
  if CP.carFingerprint in (HONDA_BOSCH | {CAR.HONDA_CIVIC, CAR.HONDA_ODYSSEY, CAR.HONDA_ODYSSEY_CHN})
```

Not needed if you aren't setting this

Reply:
Honda n-box use BOSCH ADAS, but doesn't have EPB.
Original condition code regard BOSCH Platform as using EPB, but actually not.

Reply:
Are you sure it doesn't have an EPB? Is it that the CAN message is not present? Have you checked other CAN ID's for the EPB status?

Reply:
At least N-Box has no EPB(later model 2022-- has it). And also I checked sending "450 EPB_STATUS" was failed. // Sorry I have no certain memory because this is before half year.
Do I need to check no EPB can in the car?

Reply:
How to check this. Do I need disassemble the car and confirm mechanical device is in car?
in that case I need help because I'm no familiar to car. (Honda N-Box is my first owned car)

Reply:
I cannot find epb message from can messages.